controllers/users.js

```diff
@@ -20,22 +22,27 @@ module.exports.getAllUsers = (req, res) => {
 };
 
 module.exports.createUser = (req, res) => {
-  const { name, about, avatar, email, password } = req.body;
+  const {
+
+    name, about, avatar, email, password,
+  } = req.body;
   if (password.length > 11) {
     bcrypt.hash(password, 10)
-      .then((hash) => User.create({name, about, avatar, email, password: hash}))
-      .then(({name, about, avatar, email}) => res.send({name, about, avatar, email }))
-      .catch(() => res.status(404).send({ message: 'Не удалось создать пользователя' }));
+      .then((hash) => User.create({
+        name, about, avatar, email, password: hash,
+      }))
+      .then((user) => res.send({ data: user.omitPrivate() }))
+      .catch(() => res.status(500).send({ message: 'Не удалось создать пользователя' }));
```

Comment:
Нужно исправить: при ошибки во время регистрации пользователя возвращается 500 без обработки ошибки централизованным обработчиком.
Здесь нужно точно также использовать `.catch()` вместе с `next` с передачей ошибки:
```suggestion
      .catch(() => next(new InternalServerError('Не удалось создать пользователя')));
```

Только ошибки `InternalServerError` нет, её нужно создать.